src/OpenApi/src/Services/OpenApiDocumentService.cs

```diff
@@ -268,6 +268,16 @@ private async Task<Dictionary<OperationType, OpenApiOperation>> GetOperationsAsy
                 var transformer = operationTransformers[i];
                 await transformer.TransformAsync(operation, operationContext, cancellationToken);
             }
+
+            // Apply any endpoint-specific operation transformers registered via
+            // the AddOpenApiOperationTransformer extension method.
+            var endpointOperationTransformer = description.ActionDescriptor.EndpointMetadata
+                .OfType<DelegateOpenApiOperationTransformer>()
+                .LastOrDefault();
```

Comment:
Why only use the last one and not all?

Reply:
Our _usual_ convention for metadata-like things is last-one wins, but you make a good point that this should be additive to match the rest of the transformer pattern and to allow for composition.